hugr-core/src/export.rs

```diff
@@ -1,18 +1,23 @@
 //! Exporting HUGR graphs to their `hugr-model` representation.
 use crate::{
     extension::ExtensionSet,
-    ops::OpType,
+    hugr::IdentList,
+    ops::{DataflowBlock, OpTrait, OpType},
     types::{
         type_param::{TypeArgVariable, TypeParam},
         type_row::TypeRowBase,
-        CustomType, EdgeKind, FuncTypeBase, MaybeRV, PolyFuncTypeBase, RowVariable, SumType,
-        TypeArg, TypeBase, TypeEnum,
+        CustomType, FuncTypeBase, MaybeRV, PolyFuncTypeBase, RowVariable, SumType, TypeArg,
+        TypeBase, TypeEnum,
     },
-    Direction, Hugr, HugrView, IncomingPort, Node, Port, PortIndex,
+    Direction, Hugr, HugrView, IncomingPort, Node, Port,
 };
-use bumpalo::{collections::Vec as BumpVec, Bump};
+use bumpalo::{collections::String as BumpString, collections::Vec as BumpVec, Bump};
+use fxhash::FxHashMap;
 use hugr_model::v0::{self as model};
 use indexmap::IndexSet;
```

Comment:
is this still used?

Reply:
All of them are used in that file, yes. Maybe I misunderstood?

Reply:
sorry badly phased. I see you replaced one use of `IndexSet` with `FxIndexSet`, do we still need the `indexmap` dependency. These model related PRs are pulling in a lot of new dependencies, which I am a bit wary of.

Reply:
`FxIndexSet` is a type alias: `type FxIndexSet<T> = IndexSet<T, fxhash::FxBuildHasher>;`

Reply:
🤦